packages/components/src/lib/controllers/app-controller.js

```diff
@@ -1141,7 +1152,8 @@ export default class AppController {
         this._store.commit('ui/addMessage', this.api.l10n.getMsg('TEXT_NOTICE_LEXQUERY_COMPLETE'))
         break
       case LexicalQuery.resultStatus.FAILED:
-        this.showLanguageInfo(data.homonym)
+        // TODO: Can we do anything better if homonym is not available?
```

Comment:
in the future, this might be a place to prompt the user to query a word level dictionary resource if there is one (or some other linked word-level resource -- for example, it's a place we could do better right now, if we don't have a result for a homonym in Latin, we could still offer the option to search the usage examples)

Reply:
similarly we might prompt the user to create their own analysis...

Reply:
I think this would be a good extension of the functionality. I'll add a comment about that to the TODO block.